pylint_odoo/checkers/modules_odoo.py

```diff
@@ -806,8 +806,9 @@ def _check_file_not_used(self):
         module_files = set(self._get_module_files())
         referenced_files = set(self._get_manifest_referenced_files())
         for no_referenced_file in (module_files - referenced_files):
-            if not no_referenced_file.startswith('static/'):
-                self.msg_args.append((no_referenced_file,))
+            if not (no_referenced_file.startswith('static/') or
+                    no_referenced_file.startswith('test/')):
+                self.msg_args.append((no_referenced_file))
```

Comment:
@moylop260 
Only checked into method `_check_file_not_used` this method he is in charge of `W7930(file-not-used)`
Is the solution expected?